code/modules/virus2/centrifuge.dm

```diff
@@ -111,21 +125,23 @@
 	return
 
 /obj/machinery/centrifuge/Topic(href, href_list)
+
 	if(..())
 		return
 
-	if(usr) usr.set_machine(src)
+	if(usr)
+		usr.set_machine(src)
 
 	switch(href_list["action"])
 		if("antibody")
 			var/delay = general_process_time
 			var/datum/reagent/blood/B = locate(/datum/reagent/blood) in sample.reagents.reagent_list
-			if (!B)
+			if(!B)
 				say("No antibody carrier detected.")
 
 			else if(sample.reagents.has_reagent("toxins"))
 				say("Pathogen purging speed above nominal.")
-				delay = delay/2
+				delay *= 0.5
```

Comment:
I'd say `/= 2` isn't cleaner.

Also `*= 0.5` is actually 1 character longer than `/= 2`